woocommerce/woocommerce-android
# Bundle stats networking

Closes: #10966

### Why
This PR connects the product bundle stats card with real data.

### Description
This PR adds the logic to fetch the bundle stats data and display it in the Analytics Hub screen. The `fetchProductBundlesStats` from the Analytics repository is in charge of fetching all the information needed (bundle stats from the previous and the current periods and the bundle report) and sending it to the `UpdateAnalyticsHubStats` use case. This use case will be consumed by the ViewModel then, to present the card information.

### Testing instructions
#### Prerequisites
1. A store with the product bundles extension

#### Testing
1. Open the app
3. Tap on see store analytics
4. Check that the Bundles card is shown and that it displays the correct information (You can check the values against wp-admin) *Be aware that I have just found this issue with the Last-Month granularity p1712001284389679-slack-C02KUCFCSFP

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/18119390/b02fc9b6-18df-4937-a8b5-164295550518
